jupyter-jekyll.tpl

```diff
@@ -82,7 +82,7 @@ In&nbsp;[{{ cell.execution_count }}]:
 {% endblock data_text %}
 
 {% block markdowncell scoped %}
-{{ cell.source | wrap_text(80) }}
+{{ cell.source }}
```

Comment:
It seems that we do not need to wrap the markdown cells. The theme will handle that properly.